.circleci/config.yml

```diff
@@ -470,6 +470,22 @@ jobs:
                     tests/pytest_tests/unit_tests/test_launch/ \
                     tests/pytest_tests/unit_tests/test_reports/
 
+      - when:
+          condition: { equal: [<<parameters.tests>>, importer] }
+          steps:
+            - run:
+                name: Run tests not owned by SDK
+                no_output_timeout: 10m
+                command: |
+                  CI_PYTEST_SPLIT_ARGS=" \
+                    --splits $CIRCLE_NODE_TOTAL \
+                    --group $(( $CIRCLE_NODE_INDEX + 1 ))" \
+                  tox run -v \
+                    -e <<#parameters.with_core>>core-<</parameters.with_core
+                      >>importer-wandb-py<<parameters.python_version_major>><<parameters.python_version_minor>> \
+                    -- \
+                    tests/pytest_tests/unit_tests/test_importers/
```

Comment:
i think you can put it in the `other` tests

Reply:
We could make all tests use `importer-wandb-py` but:
1. It feels less explicit
2. When the mlflow PR comes, it has different deps which I think will require this split out.

Basically we currently have:
```
    importer: pydantic>=2
    importer: polars
    importer: rich
    importer: filelock
```

But later we will have
```
    importer-mlflow: pydantic<2
    importer-wandb: pydantic>=2
    importer: polars
    importer: rich
    importer: filelock
```

Reply:
ok, so let's keep it as is for now.